### ISSUE TITLE
option to have full vertical Firewall pane

### ISSUE THREAD

**Issue by joey04** [gorhill/uBlock#2153]
In popup.js you have [the comment](https://github.com/gorhill/uBlock/blob/master/src/js/popup.js#L43) to not have the firewall pane dictate the height of the panel.

Could you add an option to allow the firewall to extend the full vertical height of the browser? It'd be nice to have a full view like the screenshots in #736, but not any taller than the browser itself. As you well know, some sites connect to plenty of domains even with 3p scripts & frames blocked by default.

Also a related question -- in popup.js you have toggleMinimize function with the comment that it's special display mode with no constraint. Is there a way to activate that? (I tried ctrl+shift+click button based on the code).

**Comment by gorhill** [comment]
> Is there a way to activate that? (I tried ctrl+shift+click button based on the code).

Hold Ctrl + Shift while clicking the _all_ cell.

**Comment by joey04** [comment]
Thanks for the instructions. At least it's a URL I can bookmark.

And I want to thank you for this truly excellent cross-browser extension. I wouldn't be making these suggestions unless it was a really useful tool in the first place.

**Comment by joey04** [comment]
FYI for anyone interested in remodelling your Firewall pane.

The utility of the stand-alone full vertical pane soon became limited (being a separate tab with an arbitrary tabID in the URL), but using it revealed how to quickly remove the vertical restriction (always execute the very first if condition in popup.js). Then I decided to further modify the 3 files popup.{html,js,css} to customize the Firewall pane to my liking. Definitely worth it.

The fact that uBO is a restartless Mozilla extension meant that I could iteratively make changes to the 3 files, remove uBO, then reinstall very quickly in Pale Moon. So I kept the browser open the whole time, which fortunately meant uBO reused its sqlite database every time. (Of course I saved uBO settings beforehand, but it was nice not to need them.)
